src/Components/Message/Message.scss

```diff
@@ -0,0 +1,139 @@
+@import '../../Assets/Styles/variables';
+
+body {
+    position: relative;
+    
+}
+
+.message {
+  position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translate(-50%, 0);
+    
+    &-container {
```

Comment:
BEM element is sperateby by two underscores "__". Please change this for all elements.